### DISCUSSION TITLE
2.0性能加载问题，300个节点，900条线，全部渲染完花费20s-25s，该怎么优化

### DISCUSSION THREAD

**Discussion by hy0913** [antvis/X6#3806] [Q&A]
看官网https://x6.antv.antgroup.com/demos/performance/index.html这里的示例，明明没这么慢，不知道为啥为加载这么久，有大佬指点下该怎么优化吗，有没有可能是父子节点关系多导致的，还是有其他原因

**Comment by NewByVector** [comment]
可以提供一个可以复现的 demo 给我们分析一下。
